papaparse.js

```diff
@@ -513,6 +511,7 @@ License: MIT
 			// First chunk pre-processing
 			if (this.isFirstChunk && isFunction(this._config.beforeFirstChunk))
 			{
+				chunk = stripBom(chunk);
```

Comment:
This only be called if befforeFisrtChunk is set, but we want to always strip bom. Could you fix it?

Reply:
you are right. this is done

Reply:
@pokoli can I get another review?